### ISSUE TITLE
Future of Uzume

### ISSUE THREAD

**Issue by 7h3730B** [dev-schueppchen/bot#17]
## Describe your issue.

We don't have many guild members who are contributing to this Project.
So it is basically dead right now.

## Describe what you were expecting instead.

Maybe we can get this to life again when using another programming language.
I would suggest node js, because lots of people on the guild are using it.

## Additional context.

For the base we could use my Project [Boto](https://github.com/7h3730B/Boto) then we are going with node and get the message handler or other stuff from it.

**Comment by zekroTJA** [comment]
Yeah sure, but I guess, the project would also be dead if we don't create an active team of people who can feel responsible for this projekt and also may promote and show new features about this project to motivate people to contribute.

And yes, a lot of people may know how to write stuff for Node / with JavaScript. The thing is, that, especially people which are new to JS and coding in general, often don't have that much knowledge about general conventions and "Do's/Don'ts" when using JS. 

Here are some proposals how we should manage the project to prevent this:
1. We should use [TypeScript](https://www.typescriptlang.org/) instead of JS. Yes, it adds another layer of tool-chaning but makes the whole code base more reliable and understandable for contributors and it can be simpified by using NPM build scripts in the `package.json`.
2. We need to force contributors to use tools like [ESLint/TSLint](https://eslint.org/). This can be enforced by using CI pipelines checking the code quality with these tools.
3. As same as this, we should also enforce the usage of a formatter like [Prettier](https://prettier.io), which should also be checked by a CI pipeline.

But in general, I think this is a good idea. In my opinion, [discord.js](https://discord.js.org) is still one of the best documented and maintained wrappers for the Discord API and also node provides a huge base of packages which we can use and which people might be comfortable with *(i.e. express)*.
